adwords_client/adwords_api/common.py

```diff
@@ -151,7 +165,7 @@ def get(self, operation, client_id=None):
             operation = operation[0]
         else:
             raise RuntimeError('The operation object is empty')
-        yield from self.ResultProcessor(self.service.get, operation)
+        return self.ResultProcessor(self.service.get, operation)
```

Comment:
Não é mais um iterator?

Reply:
Nope, but the object returned is still an iterable, so there is no problem changing it here.